client-app/shared/account/composables/useUser.ts

```diff
@@ -322,9 +334,51 @@ export function useUser() {
     }
   }
 
+  async function switchOrganization(organizationId: string): Promise<void> {
+    loading.value = true;
+
+    try {
+      await refresh(organizationId);
+
+      broadcast.emit(reloadAndOpenMainPage, null, TabsType.ALL);
+
+      /*const { error, data } = await useFetch("/connect/token")
```

Comment:
Not clear enough - will we need this part of code or it can be deleted completely?